### PR TITLE
Fyle-37drn5 fixed bug : Expenses going to draft with default vendor

### PR DESCRIPTION
**Pull request by iravisankar**

### CODE REVIEW

File: src/app/fyle/my-profile/my-profile.page.html [hunk 1]
```diff
@@ -123,7 +123,7 @@
         <ion-toggle mode="ios" class="my-profile--toggle-value" color="success"
           [(ngModel)]="orgUserSettings.currency_settings.enabled" (ionChange)="toggleCurrencySettings()"></ion-toggle>
       </div>
-      <div (click)="openCurrenySelectionModal()" class="my-profile--select-currency">
+      <div (click)="openCurrenySelectionModal()" class="my-profile--select-currency" *ngIf="orgUserSettings.currency_settings.enabled">
```
Comment (iamtarun1993) [inline src/app/fyle/my-profile/my-profile.page.html:126]: Why this change in present here?

File: src/app/fyle/add-edit-expense/add-edit-expense.page.ts [hunk 2]
```diff
@@ -1478,6 +1478,10 @@ export class AddEditExpensePage implements OnInit {
             if (expenseFieldsMap[tfc].options && expenseFieldsMap[tfc].options.length > 0) {
```
Comment (iamtarun1993) [inline src/app/fyle/add-edit-expense/add-edit-expense.page.ts:1478]: Can you add 2 screenshots of the API payload before and after changes?

Comment (iravisankar) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.ts:1478]: We get a response for Vendors like this below from online or offline.
I just set the this.fg.value.vendor_id value as default value received from payload


file

Comment (iamtarun1993) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.ts:1478]: This is the wrong screenshot, sir.
attach 2 screenshots where 
1. Vendor is missing (old one)
2. vendor is present (After you fixed)

Comment (iravisankar) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.ts:1478]: Please check the before and after screenshot

**Before**
before-fix

**After**
after-fix

**Review by iamtarun1993 — APPROVED** [review]

File: src/app/fyle/add-edit-expense/add-edit-expense.page.ts (hunk 2 above)
Comment (Dimple16) [reply src/app/fyle/add-edit-expense/add-edit-expense.page.ts:1478]: In edit case, though the `vendor_id` is present, the field is blank

**Review by Dimple16 — APPROVED** [review]

**Review by abhishek1234321 — APPROVED** [review]
